src/module-elasticsuite-catalog-optimizer/view/adminhtml/ui_component/smile_elasticsuite_catalog_optimizer_duplicate.xml

```diff
@@ -0,0 +1,793 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There should not be any new page layout for duplicate action, it's the same as the edit page.